webapp/src/ts/services/hydrate-messages.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import * as _ from 'lodash-es';
```

Comment:
Can we be specific and only import functions we use? That way tree-shaking will drop unused functions and save some bundle size automatically.

Reply:
Sure, we can try: `import { find as _find, map as _map } from 'lodash-es';`
prefixing with underscore `_find` so that we know this is coming from lodash, specially because `find` and `map` are very generic names.